### ISSUE TITLE
Level selection like original rom?

### ISSUE THREAD

**Issue by Trihy** [seraphmiii/smb-definitive#1]
Hi. Could you add level selection when finishing the game like original rom has?

Original rom let you select world at title screen with B button after finishing the game.

Would be nice to have that. Or also A button could select sub world too.

I mean this https://www.reddit.com/media?url=https%3A%2F%2Fi.redd.it%2F5dgun7knjg831.jpg&rdt=35482

Thanks.

**Comment by simplistic6502** [comment]
Yeah, that's something we'll get around to. Plan is to introduce a Zelda-esque save file system; level select will be enabled on a completed file. Thanks for the suggestion!
